### ISSUE TITLE
[bug] UX dot mobile hamburger menu accessibility for sighted keyboard users

### ISSUE THREAD

**Issue by nikkimk** [RedHat-UX/red-hat-design-system#1328]
### Describe the bug

- Sighted keyboard users have no visual indicator when the hamburger menu is focused
- While screenreader keyboard users will know it is a checkbox and will toggle the checkbox with spacebar, sighted keyboard users will see a button and expect it to work like a button on "Enter" or spacebar.

### Which Component?

ux.redhat.com

### Steps to reproduce

1. Go to https://ux.redhat.com/
2. Resize to a mobile breakpoint.
3. Tab until the mobile menu toggle is focused.
4. Click Enter.

### Expected behaviour

The "mobile toggle button" should have a focus indicator and "Enter" should be able to toggle menu just like the spacebar.

### Operating System (OS)

macOS

### Browser

Chrome

**Comment by hellogreg** [comment]
Already resolved
